recipes/mcap/all/conandata.yml

```diff
@@ -1,4 +1,10 @@
 sources:
+  "2.1.1":
+    url: "https://github.com/foxglove/mcap/archive/refs/tags/releases/cpp/v2.1.1.tar.gz"
+    sha256: "f5ec006beccfb6d56a26fee8e460e79bed02f125eb2604159ddfbb9906d57da4"
+  "2.1.0":
```

Comment:
We should only keep the newer version unless there's a specific version to add the older one :)